holmes/core/conversations_worker/realtime_manager.py

```
@@ -0,0 +1,661 @@
+                err,
+            )
+            status_str = str(status).upper()
+            if "SUBSCRIBED" in status_str:
+                self._connected = True
+                subscribed.set()
+                # Trigger a claim to cover any missed events during subscription
+                # setup or reconnects
+                try:
+                    self.on_new_pending()
+                except Exception:
+                    pass
```

Ruff (0.15.10)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)